src/reactions/index.ts

```diff
@@ -0,0 +1,163 @@
+/*
+Copyright 2024 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only
+Please see LICENSE in the repository root for full details.
+*/
+
+import { RelationType } from "matrix-js-sdk/src/types";
+
+import catSoundOgg from "../sound/reactions/cat.ogg?url";
+import catSoundMp3 from "../sound/reactions/cat.mp3?url";
+import clapSoundOgg from "../sound/reactions/clap.ogg?url";
+import clapSoundMp3 from "../sound/reactions/clap.mp3?url";
+import cricketsSoundOgg from "../sound/reactions/crickets.ogg?url";
+import cricketsSoundMp3 from "../sound/reactions/crickets.mp3?url";
+import dogSoundOgg from "../sound/reactions/dog.ogg?url";
+import dogSoundMp3 from "../sound/reactions/dog.mp3?url";
+import genericSoundOgg from "../sound/reactions/generic.ogg?url";
+import genericSoundMp3 from "../sound/reactions/generic.mp3?url";
+import lightbulbSoundOgg from "../sound/reactions/lightbulb.ogg?url";
+import lightbulbSoundMp3 from "../sound/reactions/lightbulb.mp3?url";
+import partySoundOgg from "../sound/reactions/party.ogg?url";
+import partySoundMp3 from "../sound/reactions/party.mp3?url";
+import deerSoundOgg from "../sound/reactions/deer.ogg?url";
+import deerSoundMp3 from "../sound/reactions/deer.mp3?url";
+import rockSoundOgg from "../sound/reactions/rock.ogg?url";
+import rockSoundMp3 from "../sound/reactions/rock.mp3?url";
+
+export const ElementCallReactionEventType = "io.element.call.reaction";
+
+export interface ReactionOption {
+  emoji: string;
+  name: string;
+  alias?: string[];
+  sound?: {
+    mp3?: string;
+    ogg: string;
+  };
+}
+
+export interface ECallReactionEventContent {
+  "m.relates_to": {
+    rel_type: RelationType.Reference;
+    event_id: string;
+  };
+  emoji: string;
+  name: string;
+}
+
+export const GenericReaction: ReactionOption = {
+  name: "generic",
+  emoji: "", // Filled in by user
+  sound: {
+    mp3: genericSoundMp3,
+    ogg: genericSoundOgg,
+  },
+};
+
+// The first 6 reactions are always visible.
+export const ReactionSet: ReactionOption[] = [
+  {
+    emoji: "👍",
+    name: "thumbsup",
+    alias: ["+1", "yes", "thumbs up"],
```

Comment:
This should get at least a 'TODO: i18n' if we're leaving i18n to a later iteration. (And when we do get to it, I imagine sourcing data from https://emojibase.dev/ would be better than translating names and aliases 1-to-1)

Reply:
Hm, yeah I think we need to make this an improvement. I'll log a TODO and raise an issue once this is merged.